.github/workflows/cypress.yml

```
@@ -0,0 +1,21 @@
+name: Cypress E2E Tests
+
+on:
+  push:
+    branches:
+      - main
+    schedule:
+      - cron: "0 0 * * *"
+    workflow_dispatch:
```

```suggestion
  schedule:
    - cron: "0 0 * * *"
  workflow_dispatch:
```
The `schedule` and `workflow_dispatch` events should be at the same level as `push`.